.ci/test_projects_cabal.sh

```diff
@@ -13,8 +13,8 @@ for project in *.hsfiles; do
     # Build and test with Cabal
     cabal update
     cabal build
-    cabal run test-library --enable-tests
-    cabal run doctests --enable-tests
```

Comment:
The commit is titled "remove the need", but I don't see what is changed that actually accomplishes that. I just see the flag being removed from invocations and documentation. Upon experimentation, I think the flag is still needed. Did you forget to check in some change?

Reply:
Yeah.. forgot to stage the change :see_no_evil: